src/main/resources/public/js/app.js

```diff
@@ -64,6 +64,21 @@ function fire(row, column) {
         displayGameState(currModel);
         gameModel = currModel;
 
+        setTimeout( function(){
```

Comment:
I don't think you need this setTimeout. Try removing it (and leave the inner code). It should work.